pkgs/by-name/lu/lunapnr/package.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, fetchFromGitHub
+, fetchFromBitbucket
+, gcc11Stdenv
+
+  # dependencies
+, boost
+, cmake
+, eigen
+, ninja
+, opensta
+, python3
+, qt5
+}:
+
+let
+  version = "0.1.6";
+
+  tinysvgpp = fetchFromBitbucket {
+    owner = "trcwm";
+    repo = "tinysvgpp";
+    rev = "2589ed";
+    hash = "sha256-VmD1NJUPUET1LMHn6IPTLaSJ0HEk/Q158xT0+3R9LFc=";
+  };
+in
+gcc11Stdenv.mkDerivation {
+  pname = "lunapnr";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "asicsforthemasses";
+    repo = "LunaPnR";
+    rev = "v${version}";
+    hash = "sha256-Xyt+isrMmy3MaN3cv1gsx3tOM30KLnbcaWJYgKhwjRU=";
+  };
+
+  patchPhase = ''
+    cp -r ${tinysvgpp}/* contrib/tinysvgpp
+    substituteInPlace contrib/CMakeLists.txt \
+      --replace-fail 'find_package(Git REQUIRED)' ''' \
+      --replace-fail 'if(NOT EXISTS tinysvgpp/CMakeLists.txt)' 'if(FALSE)'
```

Comment:
Removed this condition because it fails even with the `tinysvgpp/CMakeLists.txt` file existing. Should I add in a comment for it? If so, where?